packages/client/src/components/ExportButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useContext, useState } from 'react';
+import { ExportResponse } from '@guardian/transcription-service-common';
+import { AuthContext } from '@/app/template';
+import Script from 'next/script';
+import { useSearchParams } from 'next/navigation';
+import { exportTranscript } from '@/services/export';
+
+const ExportButton = () => {
+	const auth = useContext(AuthContext);
+	const searchParams = useSearchParams();
+	const [docId, setDocId] = useState<string | undefined>(undefined);
```

Comment:
I think the initial `undefined` value is not needed, when there's no value, it'd be undefined;
`const [docId, setDocId] = useState<string | undefined>();`